test-utils/utils/utils.go

```diff
@@ -31,55 +33,149 @@ const (
 	KubekinsBucket = "kubernetes-jenkins"
 	// LogDir is the directory of kubekins
 	LogDir = "logs"
-	// LogDir is the directory of the pr builder jenkins
-	PRLogDir = "pr-logs"
+	// PullLogDir is the directory of the pr builder jenkins
+	PullLogDir = "pr-logs"
```

Comment:
Changed my mind partway through (it's added in this PR, just changed in the second commit).